src/openforms/tests/e2e/input_validation_base.py

```diff
@@ -159,4 +166,9 @@ def _assertBackendValidation(self, form: Form, key: str, value: JSONValue):
                 invalid_params = response.json()["invalidParams"]
                 names = [param["name"] for param in invalid_params]
                 expected_name = f"data.{key}"
-                self.assertIn(expected_name, names)
+                if self.fuzzy_match_invalid_param_names:
+                    self.assertTrue(
+                        any(name.startswith(f"{expected_name}.") for name in names)
+                    )
```

Comment:
this doesn't allow for any errors with this exact key, so you need both the `expected_name in names or any(...)` checks